### DISCUSSION TITLE
Add logging to Estimate Sessions Extension to query on using Log Analytics Workspace in case of Unexpected Actions on Estimation Sessions like DELETEn

### DISCUSSION THREAD

**Discussion by HassanFad** [microsoft/azure-boards-estimate#274] [Ideas]
Hi Team,

Thanks for making Estimation Sessions available as Extension. We really looking forward to get this as pat of ADO system itself for more logging information to query on using KUSTO. 
Can you please in the meantime add some logging to this Extension in order to be able to have more logging nfo to query on in case of unexected actions on this Estimate Sessions like DELETED Sessions or other actions? 
We are facing an issue that our previouse created Estimate Sessions were deleted and none knows why? and by whom? 
In case of more info on this request and having context on this issue, please feel free to reach out to me by email: fadilieh@delagelanden.com
Thanks in advance.

**Comment by HassanFad** [comment]
In addition to this request, we have observed that this  Estimate Session Extension has been updated on August 4th, 2023 as mentioned on this page "https://marketplace.visualstudio.com/items?itemName=ms-devlabs.estimate&targetId=74ce0c5c-f819-479e-97c9-75f6ea0b8110&utm_source=vstsproduct&utm_medium=ExtHubManageList". Could that be the reason of deletion of our previous Estimated Sessions as far as you may know? Our previous sessions are deleted between 4th and 7th august 2023 and the actual onces are created starting by August 7th, 2023.

**Comment by mortega-bb** [comment]
All of our Estimate Sessions were completely deleted
